tests/templates/kuttl/mount-dags-pvc/03-list-jars.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: list-dags-job
```

Comment:
Do you really want to keep this ? The name of the file is a copy&paste leftover.

Reply:
Corrected the file name.